pkg/util/hlc/hlc.go

```diff
@@ -197,6 +229,28 @@ func (m *HybridManualClock) Resume() {
 	m.mu.Unlock()
 }
 
+// NewClockWithSystemTimeSource creates a Clock that reads the system time. This
+// is equivalent to NewClockWithTimeSource(timeutil.SystemTimeSource,
+// maxOffset).
+//
+// A value of 0 for maxOffset means that clock skew checking, if performed on
+// this clock by RemoteClockMonitor, is disabled.
+func NewClockWithSystemTimeSource(maxOffset time.Duration) *Clock {
+	return NewClockWithTimeSource(timeutil.SystemTimeSource, maxOffset)
+}
+
+// NewClockWithTimeSource returns a Clock configured to use a specified time
+// source.
+//
+// A value of 0 for maxOffset means that clock skew checking, if performed on
+// this clock by RemoteClockMonitor, is disabled.
+func NewClockWithTimeSource(timeSource NowSource, maxOffset time.Duration) *Clock {
```

Comment:
What's with the misalignment of `TimeSource` on the functions and `NowSource` as the type name?